solr/core/src/java/org/apache/solr/handler/admin/InfoHandler.java

```diff
@@ -150,11 +156,6 @@ public SolrRequestHandler getSubHandler(String subPath) {
 
   @Override
   public Collection<Api> getApis() {
-    return singletonList(new ReqHandlerToApi(this, getSpec("node.Info")));
-  }
-
-  @Override
-  public Boolean registerV2() {
-    return Boolean.TRUE;
+    return Collections.emptyList();
```

Comment:
why is this a empty list?    Do we even need getApis???

Reply:
I need to look into this a little more.  I _think_ that this method can go away altogether here - it exists to register the v2 APIs associated with this request handler, but in this PR that registration moves over to the 'ApiRegistrar' class where other annotation-based APIs are registered.

So maybe I can remove this here, but I'll need a little more digging to see.

Reply:
I've spent a bit of time trying to understand the `ApiSupport` interface and the `getApis` method here, and now think my initial take here was wrong.

`getApis` preexists `ApiRegistrar` and is understood implicitly in some of the data structures used to register and hold RequestHandlers (e.g. `PluginBag`).  We should probably use `getApis` where possible going forward, and reserve `ApiRegistrar` for those APIs which don't have any RequestHandler equivalent.

I'll update the PR to reflect this.